static/_redirects

```diff
@@ -322,6 +324,7 @@ https://0-20-0.docs.pomerium.com/category/guides https://0-20-0.docs.pomerium.co
 # Autocert
 /docs/reference/autocert/autocert-ca /docs/reference/autocert
```

Comment:
It looks like some redirects include an ID fragment in the URL. I think maybe we should do that for all redirects that now link to a consolidated settings page (e.g. this one would be https://www.pomerium.com/docs/reference/autocert#autocert-ca).

(if we do this, let's do it in a separate PR)